addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6975,7 +6984,57 @@ msgctxt "#14110"
 msgid "Long date format"
 msgstr ""
 
-#empty strings from id 14111 to 15011
+#: Used in Confluence
+msgctxt "#14111"
+msgid "Activities"
+msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#14112"
+msgid "Enable activity logging"
+msgstr ""
+
+#: system/settings/settings.xml
```

Comment:
don't want to be the bad guy, but it would be nice if you could add contextual help string to the new labels, even if it only says "label of a setting", because it's not obvious from the used location if it's a settings label, an option label or a description.